Formula/ffms2.rb

```diff
@@ -6,7 +6,7 @@ class Ffms2 < Formula
   sha256 "b09b2aa2b1c6f87f94a0a0dd8284b3c791cbe77f0f3df57af99ddebcd15273ed"
   # The FFMS2 source is licensed under the MIT license, but its binaries
   # are licensed under the GPL because GPL components of FFmpeg are used.
-  license "GPL-2.0"
+  license ["MIT", "GPL-2.0"]
```

Comment:
The way we've been using the multiple-license syntax is for when the user is given a choice for which license to use.

That's not the case here, because parts are under `MIT` and parts are under `GPL-2.0`. We should only specify the more restrictive license (which, I think, would be `GPL-2.0`)